packages/statistics/index.rst

```diff
@@ -850,16 +850,17 @@ Can we conclude that education benefits males more than females?
 
 .. topic:: **Take home messages**
 
-   * Hypothesis testing and p-value give you the **significance** of an
-     effect / difference
+   * Hypothesis testing and the p-value give you the **significance** of an
```

Comment:
I don't think that I agree with the addition of the "the". Maybe "p-values" (plural)?